components/components/view-modes/highlighted-small/_node-highlighted-small.css

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file
+ * This file is used to style the 'highlighted-small' view mode.
+ */
+
+/*
+  Highlighted Small
+
+  This presents the 'Highlighted Small' view mode.
+
+  Markup: node--highlighted-small.html.twig
+
+  Style guide: components.view_modes.highlighted_small
+*/
+
+.node--view-mode-highlighted-small {
+  border: 1px solid #fcece7;
+}
+
+.node--view-mode-highlighted-small .node__meta {
+  padding: 1.5rem;
+}
+
+.node--view-mode-highlighted-small .node__title {
+  margin-bottom: 1rem;
```

Comment:
Font-weight should be set to normal.

Reply:
Font size is 24px(?) in the Sketch file. And 1.424em in the pdf (which is about 22px). :|

Reply:
Font weight is now set to normal. I'm going to leave the font-size at 1.424 because that's what it is in the styleguide pdf